### DISCUSSION TITLE
Just Brilliant - thank you

### DISCUSSION THREAD

**Discussion by nextlevel2023** [bb-qq/r8152#365] [Show and tell]
I just want to say after scratching my head using all 4 ports in a LACP bond and getting poor performance I have used your driver support and finally I am getting full speed through the 2.5gb interface. You are a star and you have not only helped me but I am sure a huge number of people around the world. Thank you very much.

**Comment by BudgieBird** [comment]
@nextlevel2023 Do you mind sharing what model you have and the dongle you purchased? I'm looking to move off 1Gbit as well.

**Reply by nextlevel2023** [reply]
Sure - this model works perfect. USB 2.5G Ethernet Adapter, USB 3.0 to RJ45 2.5GbE LAN Gigabit Network Adapter with 10/100/1000/2500Mbps Full Speed for MacOS, Windows, Linux, Dell XPS, Surface Pro, Laptop, Tablet, Synology NAS and PC https://amzn.eu/d/3eOjle8
